py7zr/py7zr.py

```diff
@@ -527,6 +527,7 @@ def _extract(
         targets: Optional[List[str]] = None,
         return_dict: bool = False,
         callback: Optional[ExtractCallback] = None,
+        enable_symlink: bool = False,
```

Comment:
We should  be True in default at first, then ask users to add their preference in their project.
We can change default to be False, after many users are aware of it.